### PR TITLE
Display copied delegations in bonding section

### PR DESCRIPTION
**Pull request by r-czajkowski**
The owner of the copied delegation needs to be able to add ETH to bond. Also, this implementation takes into account escrow redelegations(`TokenGrant` contract doesn't know about redelegations from escrow) and the fact that the grantee of a managed grant can be changed.

### CODE REVIEW

File: solidity/dashboard/src/services/tbtc-authorization.service.js [hunk 1]
```diff
@@ -400,27 +330,114 @@ const fetchAvailableAmount = async (web3Context, operator) => {
...
-  web3Context,
-  operatorAddress,
-  onTransactionHashCallback
+const getGranteeOperators = async () => {
+  const web3 = await Web3Loaded
+  const { defaultAccount } = web3.eth
+  const { grantContract } = await ContractsLoaded
+
+  // Fetch all grantee operators. These are not all grantee operators,
+  // since `TokenGrant` contract does not know about escrow redelagtion.
```
Comment (dimpar) [inline solidity/dashboard/src/services/tbtc-authorization.service.js:339]: `redelegation`

File: solidity/dashboard/src/services/tbtc-authorization.service.js [hunk 2]
```diff
@@ -355,6 +271,20 @@ const fetchOperatorsOf = async (web3Context, yourAddress) => {
...
+  const copiedOperatorsFromLiquidTokens = await getCopiedOperatorsFromLiquidTokens(
+    defaultAccount,
+    Array.from(operators.keys())
+  )
+  for (let i = 0; i < copiedOperatorsFromLiquidTokens.length; i++) {
+    operators.set(
+      web3Utils.toChecksumAddress(copiedOperatorsFromLiquidTokens[i]),
+      {
+        managedGrantInfo: {},
+        isWithdrawableForOperator: true,
```
Comment (dimpar) [inline solidity/dashboard/src/services/tbtc-authorization.service.js:283]: should be probably `false`, since contract will block the transaction anyway

**Comment by dimpar** [comment]
Tested that the owner of the copied delegation is now able to add ETH to bond.

File: solidity/dashboard/src/services/tbtc-authorization.service.js (hunk 1 above)
Comment (r-czajkowski) [reply solidity/dashboard/src/services/tbtc-authorization.service.js]: pls see 91e6fdb

File: solidity/dashboard/src/services/tbtc-authorization.service.js (hunk 2 above)
Comment (r-czajkowski) [reply solidity/dashboard/src/services/tbtc-authorization.service.js]: pls see 78469b8

**Review by dimpar — APPROVED** [review]

**Pull request merged**